src/app/shared/components/template/utils/parameter-list.utils.ts

```diff
@@ -68,6 +68,28 @@ const coerceMethods = {
       })
       .catch([]),
 
+  /**
+   * Convert comma-separated or space-separated list to space-separated string.
+   * Handles both formats: "a,b,c" or "a b c" -> "a b c"
+   * @param fallback Fallback value if parsing fails
+   */
+  spaceSeparatedList: (fallback: string) =>
+    z
+      .union([z.string(), z.array(z.string())])
+      .transform((v) => {
+        // Normalize to array of strings
+        const parts = Array.isArray(v) ? v : v.includes(",") ? v.split(",") : v.split(/\s+/);
+
+        // Trim, filter empty, and join with space
+        return (
+          parts
+            .map((s) => s.trim())
+            .filter((s) => s.length > 0)
+            .join(" ") || fallback
```

Comment:
nit(non-blocking): will the fallback ever be reached? I'm guessing the `parts.map` will return an empty array if empty string (and will have parse error if undefined), so don't think the fallback will be reached

We could consider using the fallback if the array length is zero

Reply:
Tidied up in https://github.com/IDEMSInternational/open-app-builder/pull/3253/changes/12fc2140bf361167ac8ad2999b10b7107ea01ace (now outdated)